alerts/resource_alerts.libsonnet

```diff
@@ -72,6 +72,32 @@
             },
             'for': '5m',
           },
+          {
+            alert: 'NodeCPUUsage',
+            annotations: {
+              message: 'Node {{ $labels.node }} has CPU usage {{ $value }} percent (which is above the configured threshold of %s percent).' % ($._config.maxNodeCpuUsagePercent),
+            },
+            expr: |||
+              (node:node_cpu_utilisation:avg1m * 100) > %(maxNodeCpuUsagePercent)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+          },
+          {
+            alert: 'NodeMemoryUsage',
+            annotations: {
+              message: 'Node {{ $labels.node }} has memory usage {{ $value }} percent (which is above the configured threshold of %s percent).' % ($._config.maxNodeMemoryUsagePercent),
+            },
+            expr: |||
+              (node:node_memory_utilisation: * 100) > %(maxNodeMemoryUsagePercent)s
+            ||| % $._config,
+            'for': '5m',
```

Comment:
5m is way to trigger happy. If we want this at all (see my general comment on the PR) this should be at least 1h.

Reply:
Thanks - your general comment sounds good. I definitely would only want this PR to be merged if we find a way to define the prom alerts so they don't fire when there isn't a problem that someone wants to be notified of. In a couple hours I'll work on making the changes you proposed and add them to this PR.